hare_test.spec

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+# build number
+%define h_build_num  %(test -n "$build_number" && echo "$build_number" || echo 1)
+
+Summary: Hare test
+Name: cortx-hare-test
```

Comment:
`cortx-hare-test` or `cortx-hare-ivt`?

Reply:
Naming convention is like 'cortx-s3-test' or 'cortx-sspl-test'
So I think it should be 'cortx-hare-test'

Reply:
Ok. Perhaps I saw `ivt` in some other pull requests today..